app/forms/questionnaire_form.py

```diff
@@ -109,7 +108,7 @@ def _get_target_total_and_currency(self, calculation, question):
         if 'value' in calculation:
             return calculation['value'], question.get('currency')
 
-        target_answer = self.schema.get_answer(calculation['answer_id'])
+        target_answer = self.schema.get_answer_with_context(calculation['answer_id'], self.metadata, self.answer_store)
```

Comment:
I think we can remove the need for the `get_answer_with_context` method here. The target answer id is `calculation['answer_id']`, and we can assume the target currency to be the same as the currencies for the inputs on the `Calculated` question. 

We should create a card to tighten up the validation of `Calculated` questions to ensure if a currencies are used they are consistent between the total question and all breakdown answers.